src/platform/online/codeforces.py

```
@@ -54,10 +54,9 @@
     @classmethod
     def _decode_api_url(cls, api: str, **kwargs) -> str:
         url = f"https://codeforces.com/api/{api}"
-        if len(kwargs) > 0:
-            payload = '&'.join([f'{key.strip("_")}={quote_plus(str(val))}'
-                                for key, val in kwargs.items()])
-            url += f"?{payload}"
+        if kwargs:
+            payload = urlencode({k.strip("_"): v for k, v in kwargs.items()}, doseq=True)
+            route = f"{url}?{payload}"
```

Ruff (0.13.3)

Local variable `route` is assigned to but never used

Remove assignment to unused variable `route`

(F841)